packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -260,7 +260,10 @@ function GlobalStylesEditorCanvasContainerLink() {
 			// Switching to any container other than revisions should
 			// redirect from the revisions screen to the root global styles screen.
 			goTo( '/' );
+		} else if ( editorCanvasContainerView === 'global-styles-css' ) {
+			goTo( '/css' );
 		}
+
```

Comment:
This is the first time I see this effect. It's a bit weird that this relies on `goTo`. 

I had an idea that I needed elsewhere too to solve this more elegantly: Allow `NavigatorProvider` to work as a controlled component. I think that's something we'll be needing anyway in the site editor to avoid race conditions when synchronizing the URL to the state. ( cc @ciampo )

(This is not for this PR)